docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -32,86 +32,110 @@ https://PROXY_ADDRESS/v1/webapi/github/
+github](../../reference/cli.mdx#tctl-sso-configure-github) for a full reference
+of flags for this command:
 
-<Tabs>
-<TabItem label="Open Source" scope="oss">
+```code
+$ tctl sso configure github \
+--id=<Var name="GITHUB-CLIENT-ID"/>\
+--secret=<Var name="GITHUB-CLIENT-SECRET"/> \
+--teams-to-roles=<Var name="ORG-NAME,GITHUB-TEAM,access,editor"/> \
+> github.yaml
+```
+
+The contents of `github.yaml` should resemble the following:
 
 ```yaml
 kind: github
-version: v3
 metadata:
-  # Connector name that will be used with `tsh --auth=github login`
   name: github
 spec:
-  # Client ID of your GitHub OAuth App
+  api_endpoint_url: ""
   client_id: <client-id>
-  # Client secret of your GitHub OAuth App
   client_secret: <client-secret>
-  # Connector display name that will be shown on the Web UI login screen
   display: GitHub
-  # Callback URL that will be called after successful authentication
+  endpoint_url: ""
   redirect_url: https://<proxy-address>/v1/webapi/github/callback
-  # Mapping of org/team memberships onto allowed roles
+  teams_to_logins: null
   teams_to_roles:
-    - organization: octocats # GitHub organization name
-      team: admins # GitHub team name within that organization
-      # Maps octocats/admins to the "access" Teleport role
+    - organization: ORG-NAME
       roles:
         - access
+        - editor
+      team: GITHUB-TEAM
+version: v3
 ```
 
-</TabItem>
-<TabItem label="Enterprise / Cloud" scope={["enterprise", "cloud"]}>
+<Details title="Self-hosting GitHub Enterprise?" scope={["enterprise", "cloud"]} opened>
+
+For self-hosted GitHub Enterprise servers, add `spec.endpoint_url` pointing to your server instance:
 
 ```yaml
-kind: github
-version: v3
-metadata:
-  # Connector name that will be used with `tsh --auth=github login`
-  name: github
+...
 spec:
-  # Client ID of your GitHub OAuth App
-  client_id: <client-id>
-  # Client secret of your GitHub OAuth App
-  client_secret: <client-secret>
-  # Connector display name that will be shown on the Web UI login screen
-  display: GitHub
-  # URL of your GitHub Enterprise Server instance (if applicable)
+  ...
   endpoint_url: https://<github-enterprise-server-address>
-  # Callback URL that will be called after successful authentication
-  redirect_url: https://<proxy-address>/v1/webapi/github/callback
-  # Mapping of org/team memberships onto allowed roles
-  teams_to_roles:
-    - organization: octocats # GitHub organization name
-      team: admins # GitHub team name within that organization
-      # Maps octocats/admins to the "access" Teleport role
-      roles:
-        - access
+  ...
 ```
 
-</TabItem>
-</Tabs>
-
-The values of `client_id`, `client_secret`, and `redirect_url` come from the
-GitHub OAuth App you created earlier.
+</Details>
 
-Teleport will request only the `read:org` OAuth scope. Read more about OAuth scopes in GitHub's documentation:
+You can optionally test the connector configuration before applying it to your cluster:
```

Comment:
I'd strongly encourage the testing to be done prior to applying the changes.

Reply:
resolved in 8c3746791ea7a82a6a6cbba16d68869762f7a451